src/cdk/testing/test-element.ts

```diff
@@ -137,6 +137,13 @@ export interface TestElement {
    */
   text(options?: TextOptions): Promise<string>;
 
+  /**
+   * Sets the text of the element to the specified value.
+   * @param text Text to be set on the element.
+   * @breaking-change 16.0.0 Will become a required method.
+   */
+  setText?(text: string): Promise<void>;
```

Comment:
I was on the fence between introducing this method and reusing the existing `setInputValue` method, but changing it to something like this:

```
if (isInput) {
  this.value = newValue;
} else if (isContentEditable) {
  this.textContent = newValue;
} else {
  throw Error('Not an input');
}
```

The advantage of the approach above is that we don't need to introduce a new method on `TestElement`, but it's also a little misleading since the name says "input" specifically. I went with a separate `setText` method, because it's more explicit and I can see it being useful in other cases.

Reply:
I agree with your decision. I think we should clarify  the JSDoc though to make the distinction very clear, maybe even providing an example w/ contenteditable